_dev/src/components/commons/popin-configured.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <b-card
+    class="mb-2 shadow-sm border-0"
+    style="max-width: 40rem"
+  >
+    <b-card-body align="center">
+      <h3 class="font-weight-600">
+        {{ $t("configuredState.title") }}
+      </h3>
+      <img
+        src="@/assets/images/configured.png"
+        alt=""
```

Comment:
@amaury-hanser is not going to like an empty `alt`!

Reply:
Well...

>Decorative images don’t add information to the content of a page. For example, the information provided by the image might already be given using adjacent text, or the image might be included to make the website more visually attractive.
> In these cases, a null (empty) alt text should be provided (alt="") so that they can be ignored by assistive technologies, such as screen readers.